scripts/check-goleak-files.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -euo pipefail
+
+NO_GOLEAK_FILE_DIRS=""
+
+# shellcheck disable=SC2048
+for dir in $*; do
+  testFiles=$(find "${dir}" -maxdepth 1 -name '*_test.go')
+  for test in ${testFiles}; do
+    if grep -q "TestMain" "${test}" | grep -q "goleak" "${test}"; then
+      break
+    elif [[ ${test} == "/" ]]; then
+      continue
+    else
+      if [ -z "${NO_GOLEAK_FILE_DIRS}" ]; then
```

Comment:
Don't think you need this if you treat the var as array. See other scripts where we do that.

Reply:
removed it!